src/sensor/ping360.h

```diff
@@ -404,16 +404,16 @@ class Ping360 : public PingSensor
         if (_autoTransmitDuration) {
             /*
              * Per firmware engineer:
-             * 1. Starting point is TxPulse in usec = ((one-way range in metres) * 4000) / (Velocity of sound in metres per second)
+             * 1. Starting point is TxPulse in usec = ((one-way range in metres) * 8000) / (Velocity of sound in metres per second)
              * 2. Then check that TxPulse is wide enough for currently selected sample interval in usec, i.e.,
-             *    if TxPulse < (1.25 * sample interval) then TxPulse = (1.25 * sample interval)
+             *    if TxPulse < (2.5 * sample interval) then TxPulse = (2.5 * sample interval)
              * 3. Perform limit checking
              */
 
             // 1
-            int autoDuration = round(4000*range()/_speed_of_sound);
+            int autoDuration = round(8000*range()/_speed_of_sound);
```

Comment:
const int autoDuration = std::max(static_cast<int>(2.5*samplePeriod()/1000), round(4000*range()/_speed_of_sound));

define / set once

Reply:
I think the logic behind auto duration is already complex in the way that it's. We don't have the knowledge behind this steps used in this code, and I would like to maintain it as simple as possible. 
Besides that, this PR only duplicates the values, it's not supposed to change the logic behind it.